pkg/graveler/retention/garbage_collection_manager.go

```diff
@@ -84,6 +84,9 @@ func (m *GarbageCollectionManager) GetRules(ctx context.Context, storageNamespac
 	if err != nil {
```

Comment:
I would document that this is when no gc rules exits and line 87 is when gc rules were deleted

Reply:
Adding a comment that an empty file means no GC rules. I don't think it matters whether they have been deleted or otherwise.